library/src/main/java/com/chrynan/chords/widget/ChordWidget.kt

```diff
@@ -369,7 +369,7 @@ class ChordWidget : View,
     private fun calculateFretNumberPositions() {
         fretNumberPoints.clear()
 
-        for (i in (chart.fretStart - 1) until chart.fretEnd) {
+        for (i in 0 until chart.fretEnd) {
```

Comment:
I think what might work is to use the difference of `chart.fretStart` and `chart.fretEnd`.
```kotlin
for (i in 0..(chart.fretEnd - chart.fretStart))
```

Reply:
Done. Tested your suggestion in your configuration above, and it works.  Also tested with C#m (special because it's barred up a few frets):


        val chart = ChordChart(4,6,stringLabels = setOf(StringLabel(6,"E"),
                StringLabel(5,"A"), StringLabel(4,"D"), StringLabel(3,"G"),
                StringLabel(2,"B"), StringLabel(1,"e")))

        val myMarkers = HashSet<ChordMarker>()
        myMarkers.add(ChordMarker.Bar(FretNumber(4), Finger.INDEX, StringNumber(1), StringNumber(5)))
        myMarkers.add(ChordMarker.Muted(StringNumber(6)))
        myMarkers.add(ChordMarker.Note(FretNumber(5), Finger.MIDDLE, StringNumber(2)))
        myMarkers.add(ChordMarker.Note(FretNumber(6), Finger.RING, StringNumber(4)))
        myMarkers.add(ChordMarker.Note(FretNumber(6), Finger.PINKY, StringNumber(3)))
        val myChord = Chord("C#m",myMarkers)

        ChordBottomSheetDialogFragment.newInstance(myChord, chart)
                .show(supportFragmentManager, null)


![Screenshot_20200328-164214_chords](https://user-images.githubusercontent.com/7533931/77834537-7802fb00-7113-11ea-98da-b68461e8c2fc.png)